Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -11,6 +11,7 @@ Rook allows creation and customization of storage pools through the custom resou
 For optimal performance, while also adding redundancy, this sample will configure Ceph to make three full copies of the data on multiple nodes.
 
 !!! note
+
     This sample requires *at least 1 OSD per node*, with each OSD located on *3 different nodes*.
```

Comment:
@travisn we have one more option here related to callouts

```md
<!-- prettier-ignore -->
!!! callout
    text
```

or possibly 

```
<!-- prettier-ignore-start -->
!!! callout
    text
<!-- prettier-ignore-end -->
```

IMO, either of these options are worse than the one where we need to format these with an extra blank line, but it's available if you have other thoughts.

Reply:
Agreed, let's just add the blank line after callouts, instead of the ignore comment